install-dev/theme/views/layout.php

```diff
@@ -18,6 +18,10 @@
     <script type="text/javascript" src="../js/jquery/jquery-3.5.1.min.js"></script>
     <script type="text/javascript" src="../js/jquery/jquery-migrate-3.1.0.min.js"></script>
     <script type="text/javascript" src="../js/jquery/plugins/jquery.chosen.js"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/sprintf/1.1.2/sprintf.min.js" integrity="sha512-pmG0OkYtZVB2EqETE5HPsEaok7sNZFfStp5rNdpHv0tGQjbt1z8Qjzhtx88/4wsttOtDwq5DZGJyKyzEe7ribg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/zxcvbn/4.4.2/zxcvbn.js" integrity="sha512-TZlMGFY9xKj38t/5m2FzJ+RM/aD5alMHDe26p0mYUMoCF5G7ibfHUQILq0qQPV3wlsnCwL+TPRNK4vIWGLOkUQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

Comment:
This library seems a bit old https://github.com/dropbox/zxcvbn maybe not maintained a lot 😱 ?

Reply:
Even if the library is old, this is the best one also recommended by Owasp.

Reply:
There was an issue about removing every CDN from the project, related to RGPD and protecting private data, could we add zxcvbn the same way we add jQuery for the installer?

Reply:
I think yes we can do it